SmartDeviceLink/SDLLockScreenPresenter.m

```diff
@@ -169,7 +174,7 @@ - (void)sdl_dismissIOS12 {
     UIWindow *appWindow = nil;
     for (UIWindow *window in windows) {
         SDLLogV(@"Checking window: %@", window);
-        if (window != self.lockWindow) {
+        if ([window.rootViewController isKindOfClass:[self.coveredRootViewController class]]) {
```

Comment:
Can you directly check equality between `window.rootViewController` and `coveredRootViewController`? Imagine if the same class (or in the case of using `isKindOfClass` a subclass) is used in multiple windows.

Reply:
This code was removed